bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -481,6 +481,28 @@ public String toString() {
     }
 
     /**
+     * Read a sequence of entries synchronously. This is the same of
+     * {@link #readEntries(long, long)  }
+     * but it lets the client to read without checking the local value of LastAddConfirmed, so that it is possibile to
+     * read entries for which the writer has not received the acknowledge yet
```

Comment:
Please document the behaviors on a few cases:

- reading entries within LAC
- reading entries beyond LAC
- reading entries (some within LAC, some beyond LAC)